services/sidecar/src/simcore_service_sidecar/db.py

```diff
@@ -49,7 +49,6 @@ def __init__(self):
     async def __aexit__(self, exc_type, exc, tb):
         self._db_engine.close()
         await self._db_engine.wait_closed()
-        assert self._db_engine.closed
```

Comment:
FYI: you can leave that assert there as a "defensive programming" approach. You can mute bandit adding `` #nosec `` at the end. 
The PYTHON_OPTIMIZE=True option in production will remove this line